annotate.el

```diff
@@ -782,30 +802,54 @@ to 'maximum-width'."
                                                   (<= (overlay-start a)
                                                       newline-position
                                                       (overlay-end a))))
-                                      (overlays-in bol eol))
+                                  (overlays-in bol eol))
                     (lambda (x y)
                       (< (overlay-end x) (overlay-end y)))))
         ;; put each annotation on its own line
```

Comment:
This comment is now outdated and should be expanded.

Reply:
Hi!

Yes, indeed it did not made much sense, i have expanded a bit. Should the rest of the code inside the annotations iteration `(dolist (ov overlays) ...)` needs comments too?

Bye!
C.